operations/research-and-development/organization/channels.md

```diff
@@ -0,0 +1,64 @@
+# Channels
+
+Channels is split into feature teams and platform teams.
+
+## Feature Teams
+
+The feature teams are responsible for designing, developing and maintaining the features of the Channels product. They are full stack and own feature development across the entire tech stack (web app, mobile, desktop and backend).
+
+These teams are flexible enough to work on whatever is highest impact. Each team does have areas of expertise and features they own but any feature team can work on any portion of the product.
+
+The teams are:
+
+### Ursa (feature team #1)
+
+Areas of expertise:
+
+* Groups
+* LDAP
+* Permissions
+* System Console
+* Compliance
+* Data Retention
+* Mobile sidebar/categories
+
+### Interstellar (feature team #2)
+
+Areas of expertise:
+
+* Apps
+* Webhooks
+* Slash Commands
+* Plugins
+
+### Apollo (feature team #3)
+
+Areas of expertise:
+
+* Posts
+* Threads
+* File Previews
+
+## Platform Teams
+
+The platform teams are responsible for building and maintaining the systems that the feature teams rely on.
+
+**Why are these teams in the Channels product vertical and not the Platform division?**
+
+Until recently Channels was the only product Mattermost had and was itself the platform. As a result of this a lot of the platform is tightly coupled to the Channels product. The server and web app parts of the platform are especially tightly coupled. The goal is to eventually move these teams, or some version of them, out of the Channels product vertical when that coupling is looser.
+
+The teams are:
+
+### Web Platform
+
+The role of the Web Platform team is to own the infrastructure behind the Web App and Desktop App while supporting other teams that are using that infrastructure.
+
+Within the web app, that means tooling (eg. build tools, testing frameworks, style checking), core systems (eg. Redux, routing), and helping maintain the core features of the app. For the areas owned, the team is also be responsible for educating other teams about those areas and helping them coordinate work that might require changes to those things.
```

Comment:
What does "maintain the core features of the app" mean? Does this include the performance of the web app? I like the "areas of expertise" added above.  Can something similar be added here as well?